backend/src/test/kotlin/fr/gouv/cacem/monitorenv/domain/mappers/PatchMissionEntityUTest.kt

```diff
@@ -130,4 +136,47 @@ class PatchMissionEntityUTest {
         // Then
         assertThat(missionEntity.endDateTimeUtc).isNull()
     }
+
+
+    @Test
+    fun `execute() should return mission with with isUnderJdp`() {
+        // Given
+        val missionEntity = MissionFixture.aMissionEntity()
```

Comment:
todo: attention tu ne testes rien ici car la valeur de `isUnderJdp` est à `false` par défaut.

```suggestion
        val missionEntity = MissionFixture.aMissionEntity(isUnderJdp = true)
```